recipes/dask-cuda/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Submitted PR ( https://github.com/rapidsai/dask-cuda/pull/574 ) to make sure this is included in packages in the future

Reply:
BTW, Grayskull downloaded this automatically.